pkg/registry/servicecatalog/binding/storage.go

```diff
@@ -94,12 +94,12 @@ func toSelectableFields(binding *servicecatalog.Binding) fields.Set {
 }
 
 // GetAttrs returns labels and fields of a given object for filtering purposes.
-func GetAttrs(obj runtime.Object) (labels.Set, fields.Set, error) {
+func GetAttrs(obj runtime.Object) (labels.Set, fields.Set, bool, error) {
```

Comment:
Looking through git history the bool is "the uninitialized flag". I do not know what that means. My guess from looking at the commit is whether the thing that we're getting has been set?

https://github.com/kubernetes/apiserver/commit/5fa08b8c5e65fd6f5637c0ff5f9aaee37eb21903